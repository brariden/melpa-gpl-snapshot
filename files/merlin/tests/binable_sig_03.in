#!/bin/bash
. functions.sh || exit 1

reset binable_sig_03.ml
package bin_prot
tell <<END
module T : sig type ('a, 'b) t with bin_io end = struct
  type ('a, 'b) t = T of 'a * 'b with bin_io
end
module T1 : Bin_prot.Binable.S2 = T
END
cmd errors
