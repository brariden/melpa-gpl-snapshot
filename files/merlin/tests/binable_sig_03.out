["return",{"cursor":{"line":1,"col":0},"marker":false}]
["return",{"result":true}]
["return",{"cursor":{"line":1,"col":0},"marker":false}]
["return",{"cursor":{"line":5,"col":0},"marker":false}]
["return",{"cursor":{"line":5,"col":0},"marker":false}]
["return",[]]
